### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by 007john124** [JunkFood02/Seal#318] [Q&A]
When ever i try to install it says app not installed
Why is it happening

**Comment by BobbyESP** [comment]
Maybe it's because you choosed the wrong architecture of your proccessor. Download the armeabi-v7a version, this should work in all the devices.

**Reply by BobbyESP** [reply]
This is the link: https://github.com/JunkFood02/Seal/releases/download/v1.4.0/Seal-1.4.0-armeabi-v7a-release.apk
